share-encryption/src/main/java/org/alfresco/encryptor/ShareStringEncryption.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * #%L
+ *  Alfresco Share Encryption
+ * %%
+ * Copyright (C) 2005 - 2020 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software. 
+ * If the software was purchased under a paid Alfresco license, the terms of 
+ * the paid license agreement will prevail.  Otherwise, the software is 
+ * provided under the following open source license terms:
+ * 
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.encryptor;
+
+public class ShareStringEncryption
+{
+
+    static final String CMD_INITKEY = "initkey";
+    static final String CMD_ENCRYPT = "encrypt";
+    static final String CMD_VALIDATE = "validate";
+
+    static final String USAGE = "USAGE : " + ShareStringEncription.class.getName() + " " + CMD_INITKEY + " | " + CMD_ENCRYPT + " | " + CMD_VALIDATE
```

Comment:
ShareStringEncription.class.getName() - typo and I guess compilation error, since the class name was changed above. A few more like this one can be found below.